api/libproject/devstart.rb

```diff
@@ -2325,6 +2325,7 @@ def migrate_database(dry_run = false)
   common = Common.new
   common.status "Migrating main database..."
   Dir.chdir("db") do
+    ENV["DB_HOST"] = nil
```

Comment:
Don't look for locally running Cloud SQL proxy. CLOUD_SQL_INSTANCE_NAME is set, so that will be used.